src/sentry/static/sentry/app/components/issueDiff/index.tsx

```diff
@@ -0,0 +1,190 @@
+import {css} from '@emotion/core';
+import React from 'react';
+import isPropValid from '@emotion/is-prop-valid';
+import styled from '@emotion/styled';
+
+import {Client} from 'app/api';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {t} from 'app/locale';
+import withApi from 'app/utils/withApi';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import getStacktraceBody from 'app/utils/getStacktraceBody';
+import SplitDiff from 'app/components/splitDiff';
+import ButtonBar from 'app/components/buttonBar';
+import Button from 'app/components/button';
+import space from 'app/styles/space';
+import {Project} from 'app/types';
+
+import renderGroupingInfo from './groupingDiff';
```

Comment:
I'd not rename the default export

Reply:
not sure what you mean, that's the name of the function